spec/support/vcr_cassettes/lighthouse/benefits_claims/documents/lighthouse_form_526_document_upload_400.yml

```diff
@@ -0,0 +1,114 @@
+---
+http_interactions:
+- request:
+    method: post
+    uri: https://sandbox-api.va.gov/oauth2/benefits-documents/system/v1/token
```

Comment:
Same thing, mask URI

Reply:
Also fixed in https://github.com/department-of-veterans-affairs/vets-api/pull/17991/commits/8ed3889a30bca07a4c54850163bfe9d8a8de6fdf